app/services/process_payment_intent.rb

```diff
@@ -11,7 +19,8 @@ def call!
     return unless valid?
 
     last_payment.update_attribute(:cvv_response_message, nil)
-    last_payment.complete!
+    OrderWorkflow.new(@order).next
```

Comment:
this new extra behavior should be reflected in this class' unit tests IMO

Reply:
As a start, I added some for the class here: 89123428c605006c45ae4f030dd588a17ed2a11c - could probably be improved upon though.